acceptance/lib/helpers/test_helper.rb

```diff
@@ -43,9 +43,10 @@ def fixture_path
 end
 
 # Returns the contents of a named fixture file, to be found in `fixture_path`.
+# This will fail if there are multiple files with over lapping names.  ie sles-12-x86, sles-12-x86_64.
 def fixture_contents(fixture)
-  fixture_file = File.join(fixture_path, "#{fixture}.txt")
-  File.read(fixture_file)
+  fixture_file = Dir.entries(fixture_path).find { |e| /#{fixture}/ =~ e }
```

Comment:
We might want to anchor the regex here, or we risk getting the wrong fixture if one fixture's name is a substring of another's.  Maybe something like:

``` ruby
fixture_file = Dir.entries(fixture_path).find { |e| /^#{fixture}\./ =~ e }
```